components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/ServiceBusInitializer.java

```diff
@@ -515,11 +515,15 @@ protected void unsetTaskService(TaskService taskService) {
             cardinality = ReferenceCardinality.MANDATORY,
             policy = ReferencePolicy.DYNAMIC)
     public void setConfigAdminService(ConfigurationAdmin configAdminService) {
-        log.debug("Set ConfigAdmin Service");
+        if (log.isDebugEnabled()) {
+            log.debug("Setting ConfigurationAdmin Service");
+        }
         ConfigurationHolder.getInstance().setConfigAdminService(configAdminService);
     }
 
     public void unsetConfigAdminService(ConfigurationAdmin configAdminService) {
-        log.debug("Unset ConfigAdmin Service");
+        if (log.isDebugEnabled()) {
```

Comment:
we don't need the if here as we are not doing any string concatenation. Same above.

Reply:
Done